server/lib/payments.js

```diff
@@ -94,37 +95,39 @@ export const createPayment = (payload) => {
       PaymentMethodId: paymentMethod.id,
       SubscriptionId: subscription && subscription.id,
       ResponseId: response && response.id
-    }));
+    }))
+    .then(processPayment);
```

Comment:
It used to be `processPayment(Sequelize, donation)` but there is no need to pass the `Sequelize` object.

It doesn't seem that the stub cares about the number of arguments, so not sure where it's failing